types/yes_no.pp

```diff
@@ -0,0 +1 @@
+type Ssh::Yes_no = Enum['yes', 'no']
```

Comment:
this is now in stdlib

Reply:
Thanks for the hint. But I am not sure if it would be wise to use that, it accepts yes/no in all possible cases (`Pattern[/\A(?i:(yes|no))\z/]`).

The man page for ssh_config says this:
`The possible keywords and their meanings are as follows (note that keywords are case-insensitive and arguments are case-sensitive)`

From the examples in the docs I understand that keywords are the option names and arguments are the values. In that case I wouldn't like to switch to stdlib as it would allow all cases.

Reply:
Maybe we should use stdlib to match and then use a function to force the correct case.

Reply:
noooo, don't fix wrong input if we can enforce correct one in first place.
Let's follow the man page; "arguments are case sensitive".